addon/components/docs-viewer/x-autogenerated-api-docs/component.js

```diff
@@ -0,0 +1,102 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import layout from './template';
+import { readOnly } from '@ember/object/computed';
+import { set as _set } from 'lodash';
+
+export default Component.extend({
+  layout,
+  tagName: '',
+  store: service(),
+
+  sections: readOnly('project.navigationIndex'),
+
+  /*
+    Autogenerated sections include "resolved types", by which we mean things like
+    Components and Helpers, as well as generic "modules", which is any other
+    public JavaScript export from this library.
+
+    These are the sections for the resolved types.
+  */
+  resolvedTypeSections: computed(function() {
+    return this.get('sections').filter(section => section.type !== 'modules');
+  }),
+
+  /*
+    Autogenerated sections include "resolved types", by which we mean things like
+    Components and Helpers, as well as generic "modules", which is any other
+    public JavaScript export from this library.
+
+    This is the index of nodes for generic modules. We transform the raw array
+    of modules that look like this
+
+    ```
+    [
+      {id: "ember-cli-addon-docs/keyboard-config", path: "modules/ember-cli-addon-docs/keyboard-config", name: "ember-cli-addon-docs/keyboard-config"}
+      {id: "ember-cli-addon-docs/router", path: "modules/ember-cli-addon-docs/router", name: "ember-cli-addon-docs/router"}
+      {id: "ember-cli-addon-docs/utils/compile-markdown", path: "modules/ember-cli-addon-docs/utils/compile-markdown", name: "ember-cli-addon-docs/utils/compile-markdown"}
+    ]
+    ```
+
+    into a nested data structure resembling the filesystem:
+
+    ```
+    {
+      name: '@ember-cli-addon-docs',
+      children: [
+        { name: 'keyboard-config', path: "modules/ember-cli-addon-docs/keyboard-config" },
+        { name: 'router', children: [], path: "modules/ember-cli-addon-docs/router"  },
+        {
+          name: 'utils',
+          children: [
+            { name: 'compile-markdown', children: [], path: "modules/ember-cli-addon-docs/utils/compile-markdown"  },
+          ]
+        },
+      ]
+    };
+    ```
+  */
+  moduleIndex: computed(function() {
+    let modules = this.get('sections').filter(section => section.type === 'modules')[0].items;
```

Comment:
Just upgraded to latest and started hitting the following issue in this line:

![Screen Shot 2019-04-09 at 3 05 19 PM](https://user-images.githubusercontent.com/230476/55838366-13e9c000-5ad9-11e9-975a-910fde79e8ce.png)

I don't have any modules for some reason.

Reply:
@josemarluedke woops! Will fix soon, thanks for the report

Reply:
@josemarluedke Try 6.10

Reply:
Yep, that's fixed in 6.10. Thank you.